### PR TITLE
Avoid adding volume share to smb.conf using 'user.smb' option

### PR DESCRIPTION
**Pull request by anoopcs9**

### CODE REVIEW

**Comment by anoopcs9** [comment]
retest this please

File: vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2 [hunk 1]
```diff
@@ -17,3 +17,12 @@ printcap name = /dev/null
 disable spoolss = yes
 printing = bsd
 show add printer wizard = no
+
+[gluster-{{ gluster_cluster_volume }}]
```
Comment (spuiuk) [inline vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2:21]: Would love to see the whole share as a variable.

We would also have to change the location using hard coded share name gluster-vol which I believe is simply the exported sharenames in test-info.yml. 
vagrant/ansible/roles/client.prep/tasks/main.yml

and to create a series of test mounts in 
vagrant/ansible/roles/client.prep/tasks/mkdir_usermounts.yml
Although I would like to remove this all together as the this is better handled in the test runner instead.

Comment (spuiuk) [reply vagrant/ansible/roles/samba-glusterfs.setup/templates/smb.conf.j2:21]: I've posted another PR #87  which affects the request I made above. 
It removes the code mentioned in 2) above.

**Comment by spuiuk** [comment]
Anoop, Can you also make the change to the file 
vagrant/ansible/roles/client.prep/tasks/main.yml

    exported_sharenames:
      - "gluster-vol"

Use the newly created samba_share variable instead.

**Comment by anoopcs9** [comment]
> Anoop, Can you also make the change to the file
> vagrant/ansible/roles/client.prep/tasks/main.yml
> 
> ```
> exported_sharenames:
>   - "gluster-vol"
> ```
> 
> Use the newly created samba_share variable instead.

Done.

**Pull request merged**